torchrl/__init__.py

```diff
@@ -88,3 +91,8 @@ def _check_for_faulty_process(processes):
         raise RuntimeError(
             "At least on process failed. Check for more infos in the log."
         )
+def seed_generator(seed, repeat):
+    MAX_SEED_VAL =  2**32 - 1
```

Comment:
if it's a global variable you can move it out of the function, otherwise I would not use caps